pastel-utility.conf

```diff
@@ -0,0 +1 @@
+{"log-level":"info","quiet":false,"workdir":"C:\\Users\\PC1\\AppData\\Roaming\\Pastel","network":"mainnet","force":true,"peers":"","pastelexecdir":"C:\\Users\\PC1\\AppData\\Roaming\\PastelWallet","nodeversion":"latest"}
```

Comment:
Please remove this file from git and add it to gitignore

Reply:
Removed